readthedocs/oauth/services/base.py

```diff
@@ -126,7 +126,7 @@ def _updater(data):
                 datetime.fromtimestamp(data['expires_at']),
             )
             token.save()
-            log.info('Updated token.', token=token)
+            log.info("Updated token.", token_id=token.pk)
```

Comment:
was this logging the value of the token? if so, we should probably delete those logs.

Reply:
yeah :(

Reply:
It's the token itself, https://github.com/pennersr/django-allauth/blob/f421e401a6cbb44ef5b1014c54b5fac102c82b00/allauth/socialaccount/models.py#L161